scripts/smoke-tests.py

```diff
@@ -0,0 +1,96 @@
+# -----------------
+# Corgi smoke tests
+# -----------------
+#
+# Smoke tests are designed to be run against live environments
+#
+# > python3 scripts/smoke-tests.py {HOST}
+#
+# if no host is provided defaults to localhost:8000
+#
+# Note- require installation of corgi-bindings:
+
+import argparse
+import logging
+import sys
+
+import corgi_bindings
+import requests
+
+# setup logging
+logger = logging.getLogger()
+logger.setLevel(logging.INFO)
+handler = logging.StreamHandler(sys.stdout)
+handler.setLevel(logging.INFO)
+formatter = logging.Formatter("%(asctime)s - %(name)s - %(levelname)s - %(message)s")
+handler.setFormatter(formatter)
+logger.addHandler(handler)
+
+# get args
+parser = argparse.ArgumentParser()
+parser.add_argument("host", nargs="?")
+args = parser.parse_args()
+
+logger.info("smoke-test: start")
+
+# simple rest api checks
+logger.info("smoke-test: simple REST-API checks")
+response = requests.get(f"https://{args.host}")
```

Comment:
Can we please remove the scheme from these urls. I had an instance running on another machine I wanted to test which did not have TLS. In other words I want to be able to run it as:

./scripts/smoke-test.py http://<my-host>:8008

Reply:
will generalise all ref with scheme